docs/developer/api-reference/enums/allocation-strategy-enum.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: allocation-strategy-enum
+title: AllocationStrategyEnum
+hide_table_of_contents: false
+---
+
+Determine the allocation strategy for the channel.
+
+    PRIORITIZE_SORTING_ORDER - the allocation is prioritized by the warehouses' sort
+    order within the channel
+
+    PRIORITIZE_HIGH_STOCK - the allocation is prioritized by the highest available
```

Comment:
Perhaps `allocate stock in a warehouse with the most stock` is more clear?

Reply:
Same here.